### DISCUSSION TITLE
Are there more debug options than "debug=True" ?

### DISCUSSION THREAD

**Discussion by rkompass** [miguelgrinberg/microdot#147] [General]
I started to explore the async example programs and found that `static_async.py` runs on my Pico W but not on the w600.
There are no error messages, however.
Are there other options than "debug=True" in `app.run()`?
If not I will have to run the tests and compare the two platforms (and perhaps others like esp).

**Comment by miguelgrinberg** [comment]
You can look for the options to `app.run()` in the [documentation](https://microdot.readthedocs.io/en/latest/api.html#microdot.Microdot.run). I don't expect you'll find anything that can help with this problem.

Does the hello.py and hello_async.py examples work? I'm trying to understand if this is a problem with static files, with asyncio or a general problem. Unfortunately I do not have a w600, I have never tested Microdot on that board myself.
